nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StopDomainCommand.java

```diff
@@ -88,11 +88,12 @@ protected void initDomain() throws CommandException {
         // NetUtils says that the getHost() --> isThisMe() rather than merely
         // checking for the magic "localhost" string.  Too risky to fool with it today.
 
-        if (programOpts.getHost().equals(CLIConstants.DEFAULT_HOSTNAME))
+        if (programOpts.getHost().equals(CLIConstants.DEFAULT_HOSTNAME)) {
```

Comment:
You need to add the comment that you have changed the file into the header like;
// Portions Copyright [2017] [Payara Foundation and/or its affiliates]